composer.json

```diff
@@ -7,6 +7,7 @@
     }
   },
   "require": {
+    "php": ">=5.6.0",
```

Comment:
Yes, I agree, 5.6 is too big jump :)
Can we use PHP 5.4, since CentOS 7 comes with 5.4 too (CentOS 6 comes with 5.3) [CentOS PHP versions] (http://distrowatch.com/table.php?distribution=centos)

PHP 5.4 has some new very useful features like: 
- Support for ***traits*** has been added. 
- Short array syntax has been added, e.g. `$a = [1, 2, 3, 4];`
- Function array dereferencing has been added, e.g. `foo()[0];`
- Class member access on instantiation has been added, e.g. `(new Foo)->bar()`
- 'callable' was implemented as a typehint in 5.4

This features are very useful and helps write code much more simple and readable. I ask without the knowledge about how widely used A2B on CentOS 6.